patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -630,11 +630,6 @@ function openMenu() {
     });
     lockPage();
 
-    // Set the nav container height to enable scrolling to the bottom.
```

Comment:
Is this an intentional change in this PR?

Reply:
For your first comment, adjusted the z-index by its positioning.  When it sticks at the top, the feedback button goes under the accordion.
<img width="307" alt="middle" src="https://user-images.githubusercontent.com/9633303/95210458-84020080-07b9-11eb-82be-5a8eae046d94.png">
When it's in the middle of the page, the feedback button lays over the accordion like it does on other accordions to be consistent.  
<img width="309" alt="top" src="https://user-images.githubusercontent.com/9633303/95210590-a7c54680-07b9-11eb-9710-cdbc774799ec.png">
When an actual design fix takes place, this sort of things would be more consistent.

Reply:
After a conversation with @tom-fleming, the removed block was put back to resolve the scrolling issue in the menu container with iPhone.  At the same time, in the same block, added another variable to set the iPhone specific menu container height to prevent sharing the same value with the overlay offset value. This change lines the top of the overlay to the bottom of the blue bar.